### DISCUSSION TITLE
fast_float v. 3.8.1 released today

### DISCUSSION THREAD

**Discussion by GaryElshaw** [transmission/transmission#4188] [General]
https://github.com/fastfloat/fast_float/releases/tag/v3.8.1

I know, right?

**Comment by Coeur** [comment]
I like the edit. :D

**Reply by GaryElshaw** [reply]
¯\_(ツ)_/¯

**Reply by Coeur** [reply]
Should we wait for that fast lib to slow down a bit before updating it? ^_^
